sdcm/tester.py

```diff
@@ -544,6 +544,17 @@ def _init_ldap_ms_ad(self):
     def _init_ldap_openldap(self):
         self.configure_ldap(node=self.localhost, use_ssl=False)
 
+    def create_role_in_scylla(self, node: BaseNode, role_name: str, is_superuser: bool = True,
+                              is_login: bool = True):
+        self.log.debug("Configuring an LDAP Role %s in Scylla DB", role_name)
+        create_role_cmd = f'CREATE ROLE IF NOT EXISTS \'{role_name}\''
+        superuser_argument = ' WITH SUPERUSER=true' if is_superuser else ' WITH SUPERUSER=false'
+        login_argument = ' AND login=true' if is_login else ' AND login=false'
+        create_role_cmd += superuser_argument + login_argument
+        if not self.params.get('prepare_saslauthd'):
+            create_role_cmd += f' AND password=\'{LDAP_PASSWORD}\''
+        node.run_cqlsh(create_role_cmd)
```

Comment:
this function should move into the new test file, and not under `sdcm/tester.py`

Reply:
fixed